src/runmode-unix-socket.c

```diff
@@ -222,21 +242,21 @@ static TmEcode UnixSocketAddPcapFile(json_t *cmd, json_t* answer, void *data)
 #else
     if(stat(filename, &st) != 0) {
 #endif /* OS_WIN32 */
-        json_object_set_new(answer, "message", json_string("File does not exist"));
+        json_object_set_new(answer, "message", json_string("filename does not exist"));
```

Comment:
What does it mean? I've come here googling meaning of the error message.

Reply:
Can you describe what the correct message would be?

Reply:
File does not exist would be right in my case. I think there is a confusion in execution brunches due to conditional compilation but not shure where exactly (my main experience is in C# so I have eyebleading reading C).